pkg/ansible/playbook/upgradeSubnetEVM.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: "{{ target }}"
+  tasks:
+    - name: download new subnet EVM release
+      shell: wget "{{ subnetEVMReleaseURL }}"
+    - name: unpack new subnet EVM release
+      shell: tar xvf "{{ subnetEVMArchive}}"
+    - name: stop node - stop avalanchego
+      shell: sudo systemctl stop avalanchego
+    - name: replace subnet EVM binary
+      shell: cp subnet-evm "{{ subnetEVMBinaryPath }}"
```

Comment:
enable this to update a list of binary paths

Reply:
maybe only pass the vm id to the command, and encapsulate the full path here

Reply:
It is not so nice to stop and start the node many times if the node has many vms. Can you use a bash for loop?